client/components/open/forms/OpenCompleteForm.vue

```diff
@@ -2,7 +2,14 @@
   <div
     v-if="form"
     class="open-complete-form"
+    :style="{ '--font-family': form.font_family }"
   >
+    <link
```

Comment:
I think we only want to keep this here for `adminPreview` mode. When not in adminPreview, instead of having this here - ideally we should have this in the `<head>`. To do this you can use `useHead` ([see this](https://github.com/JhumanJ/OpnForm/blob/18ce6c6995dc7fb2c53d3f4cbf894a9b7b15ff4d/client/pages/forms/[slug]/index.vue#L212-L226)). I think when `adminPreview` is disabled and we're seeing the live form, we should add the `useHead` in `/forms/slug/index` instead. So:
- Keep this <link> just when adminPreview true (same for the style binding)
- Add the `useHead` on the public form page (and add the style binding)